src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNICommon.cs

```diff
@@ -5,7 +5,9 @@
 using System;
 using System.Net;
 using System.Net.Security;
+using System.Security.Authentication;
```

Comment:
```suggestion
```

Reply:
Did you test it with NetStandard2.0 and 2.1? Is it not needed?

Reply:
It show up as unnecessary for NetStandard 2.0 and 2.1 as well.